app-backend/batch/src/main/scala/stacExport/v2/CampaignExport.scala

```diff
@@ -541,18 +541,12 @@ class CampaignStacExport(
   ): IO[ExportState] = {
     for {
       // make the catalog for this annotation project
-      // make the scene item for this annotation project with an s3 asset
-      scene <- AnnotationProjectDao
-        .getFirstScene(annotationProject.id)
-        .transact(xa)
-      imageryItemO <- scene.fold(
-        imageryItemFromTileLayers(
-          annotationProject,
-          inputState.exportDefinition.taskStatuses
-        )
-      )({ scene =>
-        IO.pure(imageryItemFromScene(scene, annotationProject))
```

Comment:
After this change `imageryItemFromScene` is unused, so it can be removed 🔥

Reply:
@jisantuc yeah I meant to make a note of this. I talked to aaron about whether or not we should leave it in to be used in the future. We weren't sure but I will remove it for now

Reply:
It's in version control, so we can always get it back 🕰